gnocchi/rest/__init__.py

```diff
@@ -665,6 +665,8 @@ def post(self):
         except indexer.NoSuchResource as e:
             abort(404, e)
 
+        return metrics
```

Comment:
We should not return the user input, but the values that come from the database. Also we update a resource, so we should return a resource description, not the metrics (the resource description should have the complete metrics info).

Maybe just adding 'return' at the beginning of line 655 is enough.